mlflow/models/model.py

```diff
@@ -276,6 +278,16 @@ def metadata(self) -> Optional[Dict[str, Any]]:
         """
         return self._metadata
 
+    @property
+    def model_version(self) -> Optional[int]:
```

Comment:
nit: What about renaming it to `registered_model_version`? To make it less ambiguous and be consistent with `registered_model_name` argument of log_model.